hail/src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -149,32 +150,34 @@ object VEP {
                           warn(s"No CSQ INFO field for VEP output variant ${ VariantMethods.locusAllelesToString(vepLocus, vepAlleles) }.\nVEP output: $s.")
                           null
                       }
-                      (Annotation(locus, alleles), a)
+                      Some((Annotation(locus, alleles), a))
                     case None =>
                       fatal(s"VEP output variant ${ VariantMethods.locusAllelesToString(vepLocus, vepAlleles) } not found in original variants.\nVEP output: $s")
                   }
                 } else {
                   val jv = try {
                     JsonMethods.parse(s)
                   } catch {
-                    case _: JsonParseException =>
-                      log.warn(s"vep: failed to parse json: $s")
+                    case e: JsonParseException =>
+                      log.warn(s"VEP failed to produce parsable JSON!\n  json: $s\n  error: $e")
                       null
                   }
-                  val a = JSONAnnotationImpex.importAnnotation(jv, vepSignature)
-                  val variantString = inputQuery(a).asInstanceOf[String]
-                  if (variantString == null)
-                    fatal(s"VEP generated null variant string" +
-                      s"\n  json:   $s" +
-                      s"\n  parsed: $a")
-                  val vepv@(vepLocus, vepAlleles) = variantFromInput(variantString)
-
-                  nonStarToOriginalVariant.get(vepv) match {
-                    case Some(v@(locus, alleles)) =>
-                      (Annotation(locus, alleles), a)
-                    case None =>
-                      fatal(s"VEP output variant ${ VariantMethods.locusAllelesToString(vepLocus, vepAlleles) } not found in original variants.\nVEP output: $s")
-                  }
+                  if (jv != null) {
```

Comment:
Sorry to go around on this.  You can pull the body into the try so you don't need the null checks.